CMakeLists.txt

```diff
@@ -250,22 +251,31 @@ target_include_directories(
   PUBLIC "topi/include")
 
 # Tests
-set(TEST_EXECS "")
-file(GLOB TEST_SRCS tests/cpp/*.cc)
-find_library(GTEST_LIB gtest "$ENV{GTEST_LIB}")
-
-if(GTEST_LIB)
-  foreach(__srcpath ${TEST_SRCS})
-    get_filename_component(__srcname ${__srcpath} NAME)
-    string(REPLACE ".cc" "" __execname ${__srcname})
-    add_executable(${__execname} ${__srcpath})
-    list(APPEND TEST_EXECS ${__execname})
-    target_link_libraries(${__execname}
-      tvm ${GTEST_LIB} pthread dl)
-    set_target_properties(${__execname} PROPERTIES EXCLUDE_FROM_ALL 1)
-    set_target_properties(${__execname} PROPERTIES EXCLUDE_FROM_DEFAULT_BUILD 1)
-  endforeach()
-  add_custom_target(cpptest DEPENDS ${TEST_EXECS})
+if(USE_GTEST)
+  message(STATUS "Build with Google Test support...")
+  set(TEST_EXECS "")
+  file(GLOB TEST_SRCS tests/cpp/*.cc)
+  find_path(GTEST_INCLUDE_DIR gtest/gtest.h)
+  find_library(GTEST_LIB gtest "$ENV{GTEST_LIB}")
+
+  if(GTEST_INCLUDE_DIR AND GTEST_LIB)
+    message(STATUS "WE GOTS ${GTEST_INCLUDE_DIR} AND ${GTEST_LIB}")
+    foreach(__srcpath ${TEST_SRCS})
+      get_filename_component(__srcname ${__srcpath} NAME)
+      string(REPLACE ".cc" "" __execname ${__srcname})
+      add_executable(${__execname} ${__srcpath})
+      list(APPEND TEST_EXECS ${__execname})
+      target_include_directories(${__execname} PUBLIC ${GTEST_INCLUDE_DIR})
```

Comment:
I've added this line because `clang` doesn't search in `/usr/local/include` by default, but that's where the `gtest` header is, so now we include that path.

Before adding this, I was getting a bug where the `cpptest` target would be generated by CMake, but when I ran `make cpptest`, it couldn't find `gtest/gtest.h`.